### ISSUE TITLE
Tensorflow 2.4 Support

### ISSUE THREAD

**Issue by zeyaddeeb** [vector-ai/vectorhub#14]
I'm running into an issue with tensorflow 2.4

```python
import tensorflow as tf
tf.__version__
>>> 2.4.0

from vectorhub.encoders.text.tfhub import Bert2Vec
model = Bert2Vec()
>>> 
    ValueError: Could not find matching function to call loaded from the SavedModel. Got:
      Positional arguments (3 total):
        * {'input_word_ids': <tf.Tensor 'inputs_2:0' shape=(None, 512) dtype=int32>, 'input_mask': <tf.Tensor 'inputs:0' shape=(None, 512) dtype=int32>, 'input_type_ids': <tf.Tensor 'inputs_1:0' shape=(None, 512) dtype=int32>}
        * False
        * None
      Keyword arguments: {}

    Expected these arguments to match one of the following 4 option(s):

    Option 1:
      Positional arguments (3 total):
        * [TensorSpec(shape=(None, None), dtype=tf.int32, name='inputs/0'), TensorSpec(shape=(None, None), dtype=tf.int32, name='inputs/1'), TensorSpec(shape=(None, None), dtype=tf.int32, name='inputs/2')]
        * False
        * None
      Keyword arguments: {}

    Option 2:
      Positional arguments (3 total):
        * [TensorSpec(shape=(None, None), dtype=tf.int32, name='inputs/0'), TensorSpec(shape=(None, None), dtype=tf.int32, name='inputs/1'), TensorSpec(shape=(None, None), dtype=tf.int32, name='inputs/2')]
        * True
        * None
      Keyword arguments: {}

    Option 3:
      Positional arguments (3 total):
        * [TensorSpec(shape=(None, None), dtype=tf.int32, name='input_word_ids'), TensorSpec(shape=(None, None), dtype=tf.int32, name='input_mask'), TensorSpec(shape=(None, None), dtype=tf.int32, name='input_type_ids')]
        * True
        * None
      Keyword arguments: {}

    Option 4:
      Positional arguments (3 total):
        * [TensorSpec(shape=(None, None), dtype=tf.int32, name='input_word_ids'), TensorSpec(shape=(None, None), dtype=tf.int32, name='input_mask'), TensorSpec(shape=(None, None), dtype=tf.int32, name='input_type_ids')]
        * False
        * None
      Keyword arguments: {}
```

Any suggestion on how to fix that?

**Comment by JackyKoh** [comment]
hey @zeyaddeeb whats your vectorai version?

**Comment by zeyaddeeb** [comment]
I'm running: 
```bash
tensorflow==2.5.0-dev20201107 or tf-nightly
vectorhub==1.0.6
```
My bad, tI thought I was on 2.4.0rc1

**Comment by JackyKoh** [comment]
@zeyaddeeb Thank you. the issue was likely caused by Bert versions instead
```
version_2 = 'https://tfhub.dev/tensorflow/bert_en_uncased_L-24_H-1024_A-16/2'
version_3 = 'https://tfhub.dev/tensorflow/bert_en_uncased_L-24_H-1024_A-16/3'
```
version 2 of this requires a list as input and version 3 requires a dictionary.
Just pushed a fix for this in v1.0.7, could you try it out and let me know if the issue persists.
